src/CMakeLists.txt

```diff
@@ -362,6 +362,16 @@ if (USE_AVIF)
     endif()
 endif()
 
+if (USE_HEIF)
+    find_package(libheif 1.9.0)
```

Comment:
Please drop Findlibheif.cmake. libheif provides cmake files!

`find_package(libheif 1.9.0 CONFIG)`

Reply:
@cryptomilk https://packages.debian.org/sid/amd64/libheif-dev/filelist there are no cmake files for libheif on my system.
I see that libavif-dev provides cmake files but that is not true for libheif-dev.

Reply:
Then open a Debian bug, that they should build with cmake and package them:

rpm -ql libheif-devel | grep cmake
/usr/lib64/cmake/libheif
/usr/lib64/cmake/libheif/libheif-config-relwithdebinfo.cmake
/usr/lib64/cmake/libheif/libheif-config-version.cmake
/usr/lib64/cmake/libheif/libheif-config.cmake

Reply:
They're not shipped with [mingw ](https://packages.msys2.org/package/mingw-w64-x86_64-libheif?repo=mingw64) package either (latest 1.11.0)... Is this recent/git only?

Reply:
Those files exist since November 2019.

Reply:
Ah, they're not building using cmake, [issue opened](https://github.com/msys2/MINGW-packages/issues/8494).

Reply:
As it seems some of widely used systems like Debian, Ubuntu and mingw not shipping the cmake files, would be be okay to ship the Findlibheif.cmake until this is fixed in distros?
If I change this now, none of the darktable CI pipeline systems actually build with libheif because of missing cmake files.

Reply:
Then we have unmaintained cmake code in darktable :-( At least you need to test if it works with cmake files provided by libheif.